src/include/sof/audio/component.h

```diff
@@ -544,15 +544,16 @@ struct comp_driver_info {
  * Audio component base configuration from IPC at creation.
  */
 struct comp_ipc_config {
-	uint32_t core;		/**< core we run on */
```

Comment:
please double check the patch comments, there may have some spell mistakes, recive --> receive, thos --> those?